codex/streams/storestream.nim

```diff
@@ -110,7 +110,7 @@ method readOnce*(
     if blk.isEmpty:
       zeroMem(pbytes.offset(read), readBytes)
     else:
-      copyMem(pbytes.offset(read), blk.data[blockOffset].addr, readBytes)
+      copyMem(pbytes.offset(read), blk.data[blockOffset].unsafeAddr, readBytes)
```

Comment:
Was this intentional? The [documentation](https://nim-lang.org/docs/system.html#unsafeAddr%2CT) says:
> Warning: unsafeAddr is a deprecated alias for addr, use addr instead.

Reply:
That's for versions 2.0+